File: python/bc2708.py
nJi = 0
to = 0
while True:
    line = input().split(" ")

    cmd = line[0]
    if cmd == "ABEND": break
    else:
        nPe = int(line[1])

        if cmd == "SALIDA":
            to += nPe
            nJi += 1
        elif cmd == "VUELTA":
            to -= nPe
            nJi -= 1

print(f"{to}\n{nJi}")

File: cpp/bc2708.cpp
#include <iostream>
#include <sstream>
#include <string>
#include <cmath>
using namespace std;

int main()
{
    int nPe, nJi = 0, to = 0;
    string cmd, input, token;
    
    while(true)
    {
        getline(cin, input);
        
        istringstream iss(input);
        
        iss >> token;
        
        cmd = token;
        
        if(cmd == "ABEND") break;
        
        iss >> token;
        
        nPe = stoi(token);
        
        if(cmd == "SALIDA")
        {
            nJi++;
            to += nPe;
        }
        else if(cmd == "VUELTA")
        {
            nJi--;
            to -= nPe;
        }
        
    }
    
    cout << to << "\n" << nJi << endl;
}

